api/jobset/v1alpha2/jobset_types.go

```diff
@@ -39,6 +39,13 @@ const (
 	NodeSelectorStrategyKey string = "alpha.jobset.sigs.k8s.io/node-selector"
 	NamespacedJobKey        string = "alpha.jobset.sigs.k8s.io/namespaced-job"
 	NoScheduleTaintKey      string = "alpha.jobset.sigs.k8s.io/no-schedule"
+
+	// LabelManagedBy is used to indicate the controller or entity that manages an JobSet
+	LabelManagedBy = "jobset.sigs.k8s.io/managed-by"
```

Comment:
```suggestion
	LabelManagedBy = "alpha.jobset.sigs.k8s.io/managed-by"
```

The other labels seem to be marked as alpha so following same idea.

Reply:
done